deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkProcessor.java

```diff
@@ -90,8 +87,21 @@
     documentor = new Documentor(processingEnv);
   }
 
+  private String[] getCustomAnnotations() {
```

Comment:
ideally this would be something like a lazy val in kotlin so we don't have to recompute it on each access

Reply:
I've created a variable for it and assigned it within `init`. I think it would be better for the whole class to be in Kotlin before attempting to make it a lazy val. Unless you have a more elegant suggestion?

Reply:
this seems good, thanks!